ironmon_tracker/screens/TrackerScreen.lua

```diff
@@ -20,7 +21,7 @@ TrackerScreen.Buttons = {
 			if not self:isVisible() then return end
 			local pokemon = Tracker.getViewedPokemon()
 			local pokemonID = 0
-			if pokemon ~= nil then
+			if pokemon ~= nil and PokemonData.isValid(pokemon.pokemonID) then
```

Comment:
No reason to allow these onClicks to do anything if the pokemon itself is not valid